projects.php

```diff
@@ -13,14 +13,14 @@
 try {
     $db = new DatabaseConnection('mysql:host=db; dbname=project_manager', 'root', 'password');
 
-    $projectObjectsArray = ProjectHydrator::getAllProjects($db);
+    $projectObjects = ProjectHydrator::getAllProjects($db);
 
     $arrayOfProjectsAsAssociativeArrays = [];
-    foreach ($projectObjectsArray as $projectObject) {
-        $arrayOfProjectsAsAssociativeArrays[] = $projectObject->toAssociativeArrayFewerProperties();
+    foreach ($projectObjects as $projectObject) {
+        $arrayOfProjectsAsAssociativeArrays[] = $projectObject->toAssociativeArray();
     }
 
-    $response = new Response('Sucessfully retrieved projects', $arrayOfProjectsAsAssociativeArrays);
+    $response = new Response('Successfully retrieved projects', $arrayOfProjectsAsAssociativeArrays);
```

Comment:
this change is good as per my previous comment, but you should also change `$arrayOfProjectsAsAssociativeArrays` to `$projectsAsAssociativeArrays`